test/re/frida/ClassCreationTest.java

```diff
@@ -310,6 +311,45 @@ public void derivedInterfaceCanBeImplemented() throws ClassNotFoundException, In
         assertEquals(Arrays.equals(new String[] { "tasty", "sweet" }, fruit.getTags()), true);
     }
 
+    // Issue #76/#133
+    @Test
+    public void classWithUserDefinedFieldsCanBeImplemented() throws ClassNotFoundException, InstantiationException, IllegalAccessException, IOException {
+        loadScript("var Formatter = Java.use('re.frida.Formatter');" +
+                "var UserDefinedFields = Java.registerClass({" +
+                "  name: 're.frida.SimpleFormatter'," +
```

Comment:
This name conflicts with one of the other tests. How about naming it `re.frida.StatefulFormatter` or something like that? (And renaming both variables to match.)